nodejs/week3/database.js

```diff
@@ -0,0 +1,12 @@
+const knex = require("knex")({
+  client: "mysql2",
+  connection: {
+    host: process.env.DB_HOST || "127.0.0.1",
+    port: process.env.DB_PORT || 3306,
+    user: process.env.DB_USER || "root",
+    password: process.env.DB_PASSWORD || "hackyourfuture",
+    database: process.env.DB_NAME || "hyf_node_week3_warmup",
+    multipleStatements: false,
```

Comment:
Did you fix the SQL injection by changing this line?

It says in the warmup description:

> Hint: the multipleStatements: true part in the Knex configuration indicates how you can use the vulnerability. The Knex configuration should **not be changed** though, the SQL injection should be fixed by making changes in the /api/contacts route.